lib/buildkite/test_collector/test_links_plugin/formatter.rb

```diff
@@ -43,7 +45,7 @@ def failed_example_output(example, url)
       name = example.description
       scope_name_digest = generate_scope_name_digest(scope, name)
       test_url = "#{url}/tests/#{scope_name_digest}"
-      "\x1b[31m#{%(\x1b]1339;url=#{test_url};content="#{scope} #{name}"\x07)}\x1b[0m"
+      "🔗 \x1b[37m#{%(\x1b]1339;url=#{test_url};content="#{scope} #{name}"\x07)}\x1b[m"
```

Comment:
Was it possible in the end to underline the links? I'd really love if we can implement that convention now rather than later

Reply:
Yup, we can force an underline with ANSI so the updated UI looks like this:
<img width="634" alt="Screenshot 2023-11-07 at 9 21 29 AM" src="https://github.com/buildkite/test-collector-ruby/assets/25017977/a9cbe919-1476-497d-89aa-d38fda6b12c3">